src/index.js

```diff
@@ -333,15 +333,7 @@ class DAPIClient {
   searchUsers(pattern, limit = 10, offset = 0) { return this.makeRequestToRandomDAPINode('searchUsers', { pattern, limit, offset }); }
 
   /**
-   * @param {Object} bloomFilter
-   * @param {Uint8Array|string} bloomFilter.vData - The filter itself is simply a bit
```

Comment:
I think this doc was quite good, except it said there that it can be string, which it obviously can't, so maybe bring it back and change to `Uint8Array|Array`?